### PR TITLE
Add validation for Update Results API Params

### LINKED ISSUE #561: Validate blank/null/invalid values for keys in updateResults json
Validate blank/null/invalid values for keys in updateResults json and return appropriate error messages 

For example, updateResults API returns 201 on passing blank or invalid values to memoryRSS units or memoryRSS max

### PR DESCRIPTION
**Pull request by khansaad**
This PR fixes #561 . It contains validation for the update results API's input params.

### CODE REVIEW

**Comment by chandrams** [comment]
@khansaad - As discussed, please update the expected error messages in the negative functional tests

**Review by msvinaykumar — APPROVED** [review]
LGTM

File: src/main/java/com/autotune/analyzer/utils/AnalyzerErrorConstants.java [hunk 1]
```diff
@@ -83,9 +86,12 @@ public static final class AutotuneObjectErrors {
         public static final String UNSUPPORTED_BULK_KUBERNETES = "Bulk Kubernetes objects are currently unsupported!";
         public static final String DUPLICATE_EXPERIMENT = "Experiment name already exists: ";
         public static final String WRONG_TIMESTAMP = "The Start time should precede the End time!";
-
         public static final String MEASUREMENT_DURATION_ERROR = "Interval duration cannot be less than or greater than measurement_duration by more than " + KruizeConstants.TimeConv.MEASUREMENT_DURATION_THRESHOLD_SECONDS + " seconds";
         public static final String MISSING_METRICS = "Metric data is not present for container : %s for experiment: %s. ";
+        public static final String BLANK_AGGREGATION_INFO_VALUE = " cannot be negative or blank for the metric variable: ";
```
Comment (chandrams) [inline src/main/java/com/autotune/analyzer/utils/AnalyzerErrorConstants.java:91]: Please update the tests with the expected error messages validation

**Comment by chandrams** [comment]
@khansaad - Please raise a separate PR with the test changes as this PR is closed

**Pull request merged**